example/src/App.tsx

```diff
@@ -207,7 +198,9 @@ export default function App() {
           />
         </View>
 
-        {Platform.OS === 'android' && <RenderAndroidTests />}
+        <View style={[styles.marginBottom]}>
+          <Button title="Run Tests" color={COLOR} onPress={() => tests()} />
+        </View>
```

Comment:
What do you think to hide this button on iOS Platform?

Reply:
I forgot to hide it, done in 673007e28056318239ccbf7a3a50c82d967a35b6, thanks.